deploy/helm/edot-collector/kube-stack/managed_otlp/logs-values.yaml

```diff
@@ -0,0 +1,277 @@
+        resource/k8s: # Resource attributes tailored for services within Kubernetes.
+          attributes:
+            - key: service.name # Set the service.name resource attribute based on the well-known app.kubernetes.io/name label
+              from_attribute: app.label.name
+              action: insert
+            - key: service.name # Set the service.name resource attribute based on the k8s.container.name attribute
+              from_attribute: k8s.container.name
+              action: insert
+            - key: app.label.name # Delete app.label.name attribute previously used for service.name
+              action: delete
+            - key: service.version # Set the service.version resource attribute based on the well-known app.kubernetes.io/version label
+              from_attribute: app.label.version
+              action: insert
+            - key: app.label.version # Delete app.label.version attribute previously used for service.version
+              action: delete
+        resource/cloud:
+          attributes:
+            - key: cloud.instance.id
+              from_attribute: host.id
+              action: insert
+        # [K8s Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor)
+        k8sattributes:
+          filter:
+            # Only retrieve pods running on the same node as the collector
+            node_from_env_var: OTEL_K8S_NODE_NAME
+          passthrough: false
+          pod_association:
+            # Below association takes a look at the k8s.pod.ip and k8s.pod.uid resource attributes or connection's context, and tries to match it with the pod having the same attribute.
+            - sources:
+                - from: resource_attribute
+                  name: k8s.pod.ip
+            - sources:
+                - from: resource_attribute
+                  name: k8s.pod.uid
+            - sources:
+                - from: connection
+          extract:
+            metadata:
+              - "k8s.namespace.name"
+              - "k8s.deployment.name"
+              - "k8s.replicaset.name"
+              - "k8s.statefulset.name"
+              - "k8s.daemonset.name"
+              - "k8s.cronjob.name"
+              - "k8s.job.name"
+              - "k8s.node.name"
+              - "k8s.pod.name"
+              - "k8s.pod.ip"
+              - "k8s.pod.uid"
+              - "k8s.pod.start_time"
+            labels:
+              - tag_name: app.label.name
+                key: app.kubernetes.io/name
+                from: pod
+              - tag_name: app.label.version
+                key: app.kubernetes.io/version
+                from: pod
+      receivers:
+        # [File Log Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)
+        filelog:
```

Comment:
Can't we just rely on `logsCollection` preset that is already enabled?

Related to https://github.com/elastic/elastic-agent/issues/8843

Reply:
That would be great, but there's a small configuration difference; the preset can only exclude the `daemonset` collector logs while we are excluding any container logs within the opentelemetry-kube-stack namespace.

Disabling [includeCollectorLogs](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml#L31) will collect the gateway logs, in which the debug exporter is enabled.

Reply:
That's fine for now but I think we need to revisit in general.